doc/getting-started/scopes.rst

```diff
@@ -0,0 +1,32 @@
+Scopes
+======
+
+.. TODO(diataxis)
+   - reference: library lookup
+   - howto: vendoring
+
+Whenever Dune builds anything, it does so at the level of a *Dune workspace*. A
+Dune workspace is a set of Dune projects. A typical workspace consists of a
+single Dune project.
+
+A Dune project is defined by the presence of a
+:doc:`/reference/dune-project/index` file. Each Dune project extends over the
+subtree rooted at the directory containing the
+:doc:`/reference/dune-project/index` file, excluding any nested Dune projects.
+
+The root of the workspace is defined by the outermost
+:doc:`/reference/dune-project/index` file starting from the current directory,
+or by the first :doc:`/reference/dune-workspace/index` file found when climbing
+up the directory hierachy starting from the current directory (see
+:ref:`finding-root`). It can also be specified explicitly by passing the
+``--root`` flag (see :ref:`forcing-root`).
```

Comment:
I found this paragraph a bit unclear (you say that the root is defined by A or B, but it's not clear which applies if both make sense; and I'm not sure that I understand the descriptions of A and B). In contrast, https://dune.readthedocs.io/en/stable/usage.html#finding-the-root is clear. I would less here and redirect to that part instead. (It's not clear to me why "Finding the root" is in the "Command-line Interface" page, but oh well.)